sql/core/src/test/scala/org/apache/spark/sql/CollationSuite.scala

```diff
@@ -77,9 +107,15 @@ class CollationSuite extends DatasourceV2SQLBase with AdaptiveSparkPlanHelper {
 
   test("collate function syntax with default collation set") {
     withSQLConf(SqlApiConf.DEFAULT_COLLATION -> "UTF8_LCASE") {
-      assert(sql(s"select collate('aaa', 'utf8_lcase')").schema(0).dataType ==
-        StringType("UTF8_LCASE"))
+      assert(
+        sql(s"select collate('aaa', 'utf8_lcase')").schema(0).dataType ==
+        StringType("UTF8_LCASE")
+      )
```

Comment:
we should avoid unnecessary changes

Reply:
I got skalastyle failing without clear message in which file (bad developer expirience). So what I had to do is go through all methods I changed (or added) and do formatting by scalastyle..